Header header
thyroid_ultrasound_imaging/image_data_message image_data
sensor_msgs/Image previous_image_mask
